.husky/commit-msg

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
+
+# skip hook when in CI
+[ -n "$CI" ] && exit 0
```

Comment:
info: when is `$CI` defined? Is it a default from github actions?

Reply:
Yes, it is always set to `true` when running in Github Actions [see the docs](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables).